containers/ws/release.sh

```diff
@@ -8,24 +8,52 @@ if [ -z "${RUNC:-}" ]; then
     }
 fi
 
-$RUNC build -t quay.io/cockpit/ws:release containers/ws
+# only build on host architecture per default
+if [ -z "${ARCHS:-}" ]; then
+    case $(uname -m) in
+        x86_64) ARCHS="amd64" ;;
+        arm64) ARCHS="arm64" ;;
+    esac
+fi
+for arch in $ARCHS; do
+    $RUNC buildx build --platform="linux/${arch}" -t "quay.io/cockpit/ws:release-${arch}" containers/ws
+done
 
 # smoke test
-name=ws-release
-$RUNC run --name $name -p 19999:9090 -d quay.io/cockpit/ws:release
-until curl --fail --show-error -k --head https://localhost:19999; do
-    sleep 1
+first_arch=$(echo "${ARCHS}" | cut -d " " -f 1)
+test_container=ws-release
+$RUNC run --name "${test_container}" -p 19999:9090 -d "quay.io/cockpit/ws:release-${first_arch}"
+
+for i in $(seq 1 30); do
+    curl --fail --show-error -k --head https://localhost:19999 && break
+
+    [ "${i}" -eq 30 ] && $RUNC rm -f "${test_container}" && exit 1 || sleep 1
 done
 
 # determine cockpit version
-TAG=$($RUNC exec $name cockpit-bridge --version | sed -n '/^Version:/ { s/^.*: //p }')
+version=$($RUNC exec "${test_container}" cockpit-bridge --version | sed -n 's/^Version: *//p')
+
+$RUNC rm -f "${test_container}"
 
-$RUNC rm -f $name
+version_arch_tags=""
+for arch in $ARCHS; do
+    release_arch_tag="quay.io/cockpit/ws:release-${arch}"
+    version_arch_tag="quay.io/cockpit/ws:${version}-${arch}"
+    latest_arch_tag="quay.io/cockpit/ws:latest-${arch}"
+
+    version_arch_tags="${version_arch_tags} ${version_arch_tag}"
+
+    $RUNC tag "${release_arch_tag}" "${version_arch_tag}"
```

Comment:
Hmm, on dockerhub there was some convention to put the architecture in the repo name, like `docker.io/arm64v8/ubuntu:latest`. But indeed that doesn't seem to work on quay.io; we could create a new `quay.io/cockpit/ws-arm46` repo,  but including it into the tag seems okay. </thinking_aloud>